code/game/machinery/doors/windowdoor.dm

```diff
@@ -304,6 +319,11 @@
 	if (..())
 		autoclose = FALSE
 
+/obj/machinery/door/window/unrestricted_side(mob/opener)
+	if(get_turf(opener) == loc)
+		return turn(dir,180) & unres_sides
+	return ..()
```

Comment:
This implementation is kind of weird. It should determine which side of the door the user is coming from and ignore attempting to open it from the left or right (Consider that entering from the direction that the windoor is facing)
![image](https://user-images.githubusercontent.com/26465327/208415252-f0c0b6ce-b65c-4620-abc0-0d458b9a1385.png)

Reply:
Code is no longer included in this pr